code/hud/hudsquadmsg.h

```diff
@@ -30,56 +30,55 @@
 
 class object;
 
-// defines for messages that can be sent from the player.  Defined at bitfields so that we can enable
-// and disable messages on a message by message basis
-
-#define ATTACK_TARGET_ITEM			(1<<0)
-#define DISABLE_TARGET_ITEM			(1<<1)
-#define DISARM_TARGET_ITEM			(1<<2)
-#define PROTECT_TARGET_ITEM			(1<<3)
-#define IGNORE_TARGET_ITEM			(1<<4)
-#define FORMATION_ITEM				(1<<5)
-#define COVER_ME_ITEM				(1<<6)
-#define ENGAGE_ENEMY_ITEM			(1<<7)
-#define CAPTURE_TARGET_ITEM			(1<<8)
+// defines for messages that can be sent from the player.  Indexes into Player_orders
+
+#define ATTACK_TARGET_ITEM			0
+#define DISABLE_TARGET_ITEM			1
+#define DISARM_TARGET_ITEM			2
```

Comment:
Shouldn't this start at `1`? It looks like there is at least one case where a value of `0` is used to imply no command.

Reply:
Concur, and good catch.  Previously this was defined as (1<<0) or 1.

Reply:
good point. Addressed.
Now only multi testing remains (though I am confident that it should work, given the defines applied the changes automatically to send and recieve).
How do we go about multi testing the easiest?

Reply:
Multi should be fine but I'll pull this into a test build tomorrow and make sure of it. This will be pretty easy for me to test locally so it should be quick to verify whether it works or not.